scipy/linalg/flapack_gen.pyf.src

```diff
@@ -221,6 +221,22 @@ subroutine <prefix>getrs(n,nrhs,lu,piv,b,info,trans)
 
 end subroutine <prefix>getrs
 
+subroutine <prefix>getc2(n,a,ipiv,jpiv,info)
+    ! lu,ipiv,jpiv,info = getc2(a,overwrite_a=0)
+    ! Compute an LU factorization of with complete pivoting of a general n-by-n matrix.
+    ! A = P * L * U * Q
+    threadsafe
```

Comment:
Saw this in `<prefix>getrf`. How do I know whether this particular routine is threadsafe?

Reply:
I don't exactly know if this makes any difference but @pv might have more info.

Reply:
I don't think ``threadsafe`` makes any difference.

``threadsafe`` wraps the C/FORTRAN function call with ``Py_{BEGIN,END}_ALLOW_THREADS`` to release the gil. The Cython equivalent of this is ``nogil``, but, that is already hard coded into the template

https://github.com/scipy/scipy/blob/81d2318e3a9ab172c05645e5d663979f7c594472/scipy/linalg/_generate_pyx.py#L92-L96